Games/Quantum Conundrum/Steam/script.js

```diff
@@ -0,0 +1,12 @@
+include(["Functions", "QuickScript", "SteamScript"]);
+include(["Functions", "Verbs", "msvc90"]);
+
+new SteamScript()
+    .name("Quantum Conundrum")
+    .editor("Square Enix")
+    .author("Plata")
+    .appId(200010)
+    .preInstall(function(wine, wizard) {
+        wine.msvc90();
```

Comment:
As I said in the msvc90 discussion, it does not (tried that first). No idea why, though.